CHANGELOG.md

```diff
@@ -1,3 +1,101 @@
+### citus v11.2.0 (January 30, 2023) ###
+
+* Adds support for outer joins having recurring tuples in the outer side of the
```

Comment:
I'd probably still avoid "recurring tuple" altogether from the description.

What about: `Adds support for outer joins with reference tables / complex subqueries and ctes in the outer side of the
  join (e.g., \<reference table\subquery
cte> LEFT JOIN \<distributed table\subquery\>)`